src/js/main.js

```diff
@@ -202,7 +202,8 @@ const sticker = {
 stickable.onmousedown = e => {
 	if (e.which !== 1) return; // Only work on left mouse button
 	const onSticker = e.target.classList.contains("sticker");
-	if (onSticker && !sticker.current) {
```

Comment:
Solved it by removing `!sticker.current`: 
- Places all the stickers initially around.
- Assigns the current prop on all intially generated stickers.
- Last generated sticker received a DOM element in de  "current" variable, wich was the culprit of always jumping to the else block, because it wasn't false.

@RoelN Was there a reason maybe checking on `!sticker.current`?

Reply:
This might've been a lingering piece of logic from the first implementation... I don't see what good it does except causing bugs, so good riddance!

Thanks for the cleanup!